doc/reference.rst

```diff
@@ -1896,11 +1896,12 @@ Possible values:
   8 digits match the current iso-date.
 
 .. NOTE::
-   In case of ``unixtime``, if the resulting serial is lower or equal than current zone
-   (this happens e.g. in case of migrating from other policy or frequent updates)
-   the serial is incremented instead.
+   If the serial for ``unixtime`` or ``dateserial`` is lower or equal than the
```

Comment:
What do you think?
```
   If the resulting serial for ``unixtime`` or ``dateserial`` would be lower or equal
   than the current zone's serial (this happens e.g. when migrating from other policy or
   frequent updates), the current zone's serial is incremented instead.         
                                                                                
   To avoid user confusion, use ``dateserial`` only if you expect fewer than    
   100 updates per day per zone and ``unixtime`` only if you expect at most     
   one update per second per zone.
```

Reply:
Good idea. I also changed the first part of the sentence, so that both talk about "at most" (otherwise, questions might be raised why one is `<` and the other `≤`.